docs/edgeql/select.rst

```diff
@@ -843,6 +843,48 @@ shapes just like a non-computed link.
   } filter .name = "Iron Man";
 
 
+Using backlinks to find items
+-----------------------------
+
+Another handy use for backlinks is using them to filter and find items
+such as during an update. This can work as a nice shortcut when you have
+the id of one object that links to a second object without a link back
+to the first.
+
+Spider-Man's villains always have a grudging respect for him, and their names
+can be changed to reflect that if we know the id of a movie that they starred
+in. (Note the ability to :ref:`cast <ref_eql_casts>` from a ``uuid`` to an
```

Comment:
I think it would be more useful here to link to the specific part about casting from a UUID since that's the functionality you're calling out. You might have to add a label to be able to link to it directly. (I don't think it has one yet.) If you do, I would also link the whole phrase: "cast from a uuid" (Note: you won't be able to keep `uuid` as inline code if you do this because RST doesn't want any inline styles on links.)